jolly-jellyfish/src/django_meta/settings.py

```diff
@@ -4,9 +4,14 @@
 import random
 import string
 
+
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 MEDIA_ROOT = os.path.join(BASE_DIR, 'user_media')
+MEDIA_URL = "/user_media/"
```

Comment:
```suggestion
MEDIA_URL = '/user_media/'
```
Consistency improvement.